default/AI/ShipDesignAI.py

```diff
@@ -2006,7 +2184,7 @@ def _get_design_by_name(design_name, verbose=False, update_invalid=False):
     return design
 
 
-def _get_part_type(partname):
+def get_part_type(partname):
```

Comment:
I think this changes should be ported to master. They are not related to fighter directly. If you want I can do it.